src/components/tutorial/Tutorial.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Mode } from "../game/GameState";
+import { basicButtonStyle, overlayWrapperStyle, titleStyle } from "../MenuUtils";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEnvelope } from "@fortawesome/free-solid-svg-icons";
+import { ReturnToMenu } from "../mainMenu/ReturnToMenu";
+
+const wrapperStyle = {
```

Comment:
Set the background overlay in its own class in styles.js and import it into a separate div